install.sh

```diff
@@ -2,56 +2,103 @@
 set -o nounset
 set -o pipefail
 
+class DockerComposeInstaller {
+    constructor() {
+        self.os=$(lsb_release -si)
+        self.arch=$(uname -m)
+    }
+
+    install() {
+        echo "Installiere Docker Compose für $self.os auf $self.arch"
```

Comment:
This should absolutely not be in German lol

Reply:
you're absolutly right about the german. My bad sorry!

I mean its a design decision you're taking.  I just wanted to make it more convinent since in my perpesktive an installer script takes care of the whole installation. Feel free to reject the PR

```suggestion
        echo "installing Docker Compose for $self.os on $self.arch"
```